README.md

```diff
@@ -1 +1,141 @@
-# grepg
+grepg
+===
+
+`grepg` (pronounced Grep G) is a python client for [GrepPage](https://www.greppage.com).  It allows you to access your cheat sheets without leaving the terminal.
+
+![grepg screenshot](https://github.com/tejal29/grepg/raw/master/img/screenshot.png)
+
+#Installation
+To install `grepg` run
+
+```
+pip install grepg
+```
+
+#Requirements
+- python 2.7.3 and greater
+
+
+#Usage
+
+Enter user and topic name followed by an optional search string.
+
+```
+Usage: grepg -u <user_name> [-t <topic_name> -s <search_term>]
+
+Options:
+  -h, --help, --short-help
+                        show this help message and exit.
+  --long-help           show options from all registered modules, not just the
+                        __main__ module.
+  -v, --verbose         Log to stdout [default: False]
+  -f, --force           Force fetch cheats and update cached cheats. Default
+                        Cache TTL 10 seconds [default: False]
+  -u USER, --user=USER  username
+  -t TOPIC, --topic=TOPIC
+                        topic
+  -s SEARCH, --search=SEARCH
+                        text to search
+  -c, --colorize        colorize output [default: False]
+
+Examples:
+  grepg -u tejal29
+  grepg -u tejal29 -t python
+  grepg -u tejal29 -t python -s open
+
+Defaults:
+  To set default user, create a file in ~/.grepg.yml with
+  user: test
+  ttl_in_seconds: 86400 # 1 day
+```
+
+
+##List all topics for a user
+
+```
+$ grepg -u tejal29
+User: tejal29, Last Fetched: 4 second(s) ago
+Available Topics =>
+scala	Shell	Vertica	Mac	Javascript	Python	Misc
+
+```
+
+##List all cheats for the `scala` topic for user `tejal29`
+
+```
+$ grepg -u tejal29 -t scala
+User: tejal29, Topic: scala, Last Fetched: 0 second(s) ago
+Convert List of string to Set
+‘this is a list this’.split(‘ ‘).map(Word(_)).toSet
+
+Case class example
+case class Person(firstName: String, lastName:String)
+...
+```
+
+##Search for a specific string
+
+```
+$ grepg -u tejal29 -t scala -s 'case class'
+User: tejal29, Topic: scala, Search-Term: case, Last fetched: 5 second(s) ago
+Case class example
+case class Person(firstName: String, lastName:String)
+
+Case Multiple match
+val c: Char = 'f'
+c match {
+   case ‘,’ | ‘.’ => println(‘punctuation’)
+   case _       => println(‘not punctuation’)
+}
+
+```
+
+
+#Configuration
+Setup defaults in `~/.grepg.yml`
```

Comment:
" will fetch all the cheats from until the cache is expired" should be "is the ttl for the cache."